### DISCUSSION TITLE
Azure SQL Server Connection - Error when trying to expand the schema - Using the current Beta Version 21.0.5.202105111250 on Win10

### DISCUSSION THREAD

**Discussion by davedran** [dbeaver/dbeaver#12770] [Q&A] [answered]
Hi

My connection to Data Verse via SQL Server 18 using Active directory MFA connection works well.

But, with DBeaver using Azure SQL Server connection, I get an error when I tried to expand the schema tree.

SQL Error [40000] [S0001]: Invalid table name extended_properties for schema sys. Line:2, Position:17
RequestId: TDS;3ce155b2-0ef1-4ba4-bf71-f88fc308e44e;4
Time: 2021-05-12T13:34:39.7217781Z
  Invalid table name extended_properties for schema sys. Line:2, Position:17
RequestId: TDS;3ce155b2-0ef1-4ba4-bf71-f88fc308e44e;4
Time: 2021-05-12T13:34:39.7217781Z
  Invalid table name extended_properties for schema sys. Line:2, Position:17
RequestId: TDS;3ce155b2-0ef1-4ba4-bf71-f88fc308e44e;4
Time: 2021-05-12T13:34:39.7217781Z

**Comment by ShadelessFox** [comment, marked as answer]
Hello @davedran,

Looks like your user does not have permissions for reading metadata tables.

For more information please refer to:
https://docs.microsoft.com/en-us/sql/relational-databases/security/metadata-visibility-configuration
